kong-2.0.0rc1-0.rockspec

```diff
@@ -38,8 +38,8 @@ dependencies = {
   "lua-messagepack == 0.5.2",
   "lua-resty-openssl == 0.3.0",
   -- external Kong plugins
-  "kong-plugin-azure-functions ~> 0.4",
-  "kong-plugin-zipkin ~> 0.2",
+  "kong-plugin-azure-functions ~> 0.4.2",
```

Comment:
I think maybe this should be changed to `>=` instead of `~>`, otherwise it will never use a newer version even if one exists. @hishamhm whats your suggestions?

Reply:
`~> 0.4` already does this. It means `>= 0.4.0, < 0.5.0`